spk/jellyfin/Makefile

```diff
@@ -4,10 +4,9 @@ SPK_VERS = 10.10.3
 SPK_REV = 20
```

Comment:
```suggestion
SPK_REV = 21
```

Reply:
I intentionally kept the revision.
I don't want to bother users with DSM 7.2 by getting an update with no changes.
We will publish the same revision for DSM 6.2.4 and DSM 7.1 only, leaving the DSM 7.2 packages untouched.
We can then omit the DSM 7.2 packages for future updates of Jellyfin.